sig-docs/README.md

```diff
@@ -11,7 +11,7 @@ To understand how this file is generated, see generator/README.md.
 Covers documentation, doc processes, and doc publishing for Kubernetes.
 
 ## Meetings
-* [Tuesdays at 17:30 UTC](https://zoom.us/j/678394311) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=UTC).
+* [Tuesdays at 17:30 UTC](https://zoom.us/j/678394311) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=UTC).
```

Comment:
Also modify the same thing in https://github.com/kubernetes/community/blob/master/sig-list.md please.